cmd/kyma/alpha/uninstall/cmd.go

```diff
@@ -78,10 +80,23 @@ func (cmd *command) Run() error {
 		HelmTimeoutSeconds:            int(cmd.opts.HelmTimeout.Seconds()),
 		BackoffInitialIntervalSeconds: 3,
 		BackoffMaxElapsedTimeSeconds:  60 * 5,
-		Log:                           log.Printf,
+		Log:                           cmd.getLogFunc(),
 	}
 
-	installer, err := deployment.NewDeployment(prerequisitesContent, componentsContent, nil, "path", installationCfg)
+	var updateCh chan deployment.ProcessUpdate
+	if cmd.opts.Verbose {
+		defer log.Println("Kyma uninstalled!")
+	} else {
+		// Start rendering async CLI UI
+		asyncUI := asyncui.AsyncUI{StepFactory: &cmd.Factory}
+		updateCh, err = asyncUI.Start()
+		if err != nil {
+			return err
+		}
+		defer asyncUI.Stop() // stop receiving update-events and wait until UI rendering is finished
+	}
```

Comment:
Hey, could you explain the logic behind it? Why is asyncui turned on only when the verbose flag is not set?

Reply:
Sure, the reason is that the verbose-mode is just enabling the logging of the underneath executed Hydroform-API. The CLI should then be quiet to avoid redundant information shown to end-users.